### ISSUE TITLE
Passing "/" in username field to "tfx login --authType basic" causes crash

### ISSUE THREAD

**Issue by staticcat** [Microsoft/tfs-cli#19]
The command line was:
```
tfx login --authType basic
```

Then I entered the following by mistake:

```
Enter collection url > http://localmachine:8080/tfs/DefaultCollection
Enter username > domain/user
```

Which resulted in:
```
unhandled:
TypeError: Cannot read property 'toString' of null
    at C:\Users\AppData\Roaming\npm\node_modules\tfx-cli\lib\inputs.js:70:65
    at Interface.onLine (C:\Users\AppData\Roaming\npm\node_modules\tfx-cli\node_modules\read\l
ib\read.js:111:5)
    at Interface.emit (events.js:107:17)
    at Interface._onLine (readline.js:214:10)
    at Interface._line (readline.js:553:8)
    at Interface._ttyWrite (readline.js:830:14)
    at ReadStream.onkeypress (readline.js:109:10)
    at ReadStream.emit (events.js:110:17)
    at readline.js:1175:14
    at Array.forEach (native)
```

I  would expect to report an error to standard error that the input was invalid.

**Comment by teddyward** [comment]
I can't repro this on 0.1.9 unless I abort halfway through entering creds by hitting ^C. Are you using the most recent version (published yesterday afternoon)?

Will change to make abort output more user friendly

**Comment by teddyward** [comment]
fixed d9f167d5afd44a9984d206f4afae47a7e884526d
